src/renderer/containers/map/infoBox/infoBox.scss

```diff
@@ -63,3 +63,11 @@
   text-align: center;
   padding-bottom: 2%;
 }
+
+.freshtrak-link {
```

Comment:
if these are all the same styles as the **.number** style above you can combine these styles with a comma like so `.number, .freshtrak-link {`